### PR TITLE
dm/setup_circleci

### PR DESCRIPTION
**Pull request by svintbel**
Add config files for circleci specified for changes in branch ms/setup_rails_boilerplate. Cause I can't add access for circleci, I've tested it under forked repository, so this should work here to after merging ms/setup_rails_boilerplate branch.

Screenshot from 2019-08-06 00-00-07

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+jobs:
+  build:
+   docker:
+      - image: circleci/ruby:2.6.3-node
+        environment:
+          RAILS_ENV: test
+          PGHOST: 127.0.0.1
+          PGUSER: root
+      - image: circleci/postgres:9.6.2-alpine
```
Comment (LuginaJulia) [inline .circleci/config.yml]: Change postgres version on 11.4

File: .circleci/config.yml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+        environment:
+          POSTGRES_USER: root
+          POSTGRES_DB: nicher-events_development
```
Comment (LuginaJulia) [inline .circleci/config.yml]: It is more logical: nicher-events_test

File: .circleci/config.yml [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+   working_directory: ~/nicher-events
+   steps:
+      - checkout
+      - run: |
+          yarn install --check-files
+          bundle install
+      - run: |
+          sudo apt install -y postgresql-client || true
+      - run: |
+          bundle exec rake db:create
```
Comment (LuginaJulia) [inline .circleci/config.yml]: After db:create add db:migrate

File: .circleci/config.yml [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+          bundle exec rake test
```
Comment (LuginaJulia) [inline .circleci/config.yml]: Use rspec instead of unit test

File: .circleci/config.yml (hunk 1 above)
Comment (svintbel) [reply .circleci/config.yml]: done

File: .circleci/config.yml (hunk 2 above)
Comment (svintbel) [reply .circleci/config.yml]: done

File: .circleci/config.yml (hunk 3 above)
Comment (svintbel) [reply .circleci/config.yml]: done

File: .circleci/config.yml (hunk 4 above)
Comment (svintbel) [reply .circleci/config.yml]: done

**Comment by anatoliliotych** [comment]
bundler: command not found: rspec

**Comment by svintbel** [comment]
hmm, cause in staging there now rspec in gemfile

**Comment by svintbel** [comment]
Yeap, Rspec added to Gemfile only in PR#2, i think no problems will be with it after merging all into one branch/

**Comment by anatoliliotych** [comment]
you can't pass circle ci without rspec, please add it in your branch.

**Pull request merged**